src/TestInfrastructure/Framework/TestLogger.cs

```diff
@@ -0,0 +1,57 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    public class TestLogger : ILogger
+    {
+        private IList<string> outputStrings = new List<string>();
+
+        public void AssertOutputStrings(int expectedOutputMessages)
+        {
+            this.outputStrings.Should().HaveCount(expectedOutputMessages);
+        }
+
+        public void AssertOutputStrings(params string[] orderedOutputMessages)
```

Comment:
There wasn't previously a simple implementation of _ILogger_ - the existing tests wrote used the _ConfigurableOutputWindowPane_ class (which is where I copied the assertions from).